aziotctl/aziotctl-common/src/restart.rs

```diff
@@ -0,0 +1,55 @@
+use std::io::{self, Write};
+use std::process::Command;
+
+use anyhow::{Context, Result};
+
+use crate::ServiceDefinition;
+
+#[allow(clippy::missing_errors_doc)]
```

Comment:
Like I said last time:

>Just `#![allow]` the clippy lint that told you to add them. (See `CONTRIBUTING.md` for the list of clippy lints it's okay to allow in general, and why this lint ought to be allowed in particular.)

The `!` in `#!` was to point that you should allow it globally, in lib.rs